reactor-core/src/main/java/reactor/core/publisher/DirectProcessor.java

```diff
@@ -79,7 +79,8 @@
  * </p>
  *
  * @param <T> the input and output value type
- * @deprecated Prefer clear cut usage of {@link Sinks}, to be removed in 3.5
+ * @deprecated To be removed in 3.5, prefer clear cut usage of {@link Sinks}:
+ * <pre>Sinks.empty();</pre> for example.
```

Comment:
I think `Sinks.empty()` isn't a valid replacement for `DirectProcessor` since it can only signal error or complete.

Although I am not sure what would be the replacement :D Let me have a look and get back to you...

Reply:
I wasn't sure neither, thanks for your help 😅

Reply:
`DirectProcessor` is `Sinks.many().multicast().onBackpressureError()`